clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobStreamer.java

```diff
@@ -34,10 +36,20 @@ static JobStreamer noop() {
   @ThreadSafe
   final class NoopJobStream implements JobStreamer {
     private static final NoopJobStream INSTANCE = new NoopJobStream();
+    private static boolean isOpen = false;
+
+    public void setIsOpen(final boolean isOpen) {
+      this.isOpen = isOpen;
+    }
 
     @Override
     public void close() {}
 
+    @Override
+    public boolean isOpen() {
```

Comment:
:thought_balloon: Hm, I get the idea - what we want to know is, is streaming enabled? I'm wondering if it makes sense to add it here. The whole point of having the interface was that you don't really need to know this. In this sense, it's not really a concern of the streamer if we want to back off or not, but a job worker issue. But I don't have a good idea, so let's leave it like that for now. You could make it final however, I don't think we need the `setIsOpen`.

Reply:
Sorry, my bad, I meant the field can be final